x-pack/plugins/observability_solution/observability/server/services/slo/get_preview_data.ts

```diff
@@ -461,6 +464,98 @@ export class GetPreviewData {
     }
   }
 
+  private async getSyntheticsAvailabilityPreviewData(
+    indicator: SyntheticsAvailabilityIndicator,
+    options: Options
+  ): Promise<GetPreviewDataResponse> {
+    const filter = [];
+    const { monitorIds, tags, projects } = buildParamValues({
+      monitorIds: indicator.params.monitorIds || [],
+      tags: indicator.params.tags || [],
+      projects: indicator.params.projects || [],
+    });
+    if (!monitorIds.includes(ALL_VALUE) && monitorIds.length > 0)
+      filter.push({
+        terms: { 'monitor.id': monitorIds },
+      });
+    if (!tags.includes(ALL_VALUE) && tags.length > 0)
+      filter.push({
+        terms: { tags },
+      });
+    if (!projects.includes(ALL_VALUE) && projects.length > 0)
+      filter.push({
+        terms: { 'monitor.project.id': projects },
+      });
+
+    const result = await this.esClient.search({
+      index: SYNTHETICS_INDEX_PATTERN,
+      size: 0,
+      query: {
+        bool: {
+          filter: [
+            { range: { '@timestamp': { gte: options.range.start, lte: options.range.end } } },
+            { exists: { field: 'summary.final_attempt' } },
+            { term: { 'meta.space_id': this.spaceId } },
+            ...filter,
+          ],
+        },
+      },
+      aggs: {
+        perMinute: {
+          date_histogram: {
+            field: '@timestamp',
+            fixed_interval: '10m',
+          },
+          aggs: {
+            good: {
+              filter: {
+                range: {
+                  'summary.up': {
+                    gte: 1,
+                  },
+                },
+              },
+            },
+            bad: {
+              filter: {
+                range: {
+                  'summary.up': {
+                    lte: 0,
+                  },
+                },
```

Comment:
We are not using the same aggregation here and in the transform. Is it on purpose?